plugins.yaml

```diff
@@ -342,7 +342,7 @@ aiida-siesta:
   documentation_url: https://aiida-siesta-plugin.readthedocs.io/
   entry_point_prefix: siesta
   pip_url: aiida-siesta
-  plugin_info: https://raw.github.com/siesta-project/aiida_siesta_plugin/master/setup.json
+  plugin_info: https://raw.githubusercontent.com/siesta-project/aiida_siesta_plugin/develop/pyproject.toml
```

Comment:
I would prefer this points to 'master' branch since it is the one corresponding to the latest release and it seems the customary for all the other plugins.

Reply:
Yep, thanks!